configuration/go/sdk/config-subscriber/app.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	dapr "github.com/dapr/go-sdk/client"
+)
+
+func main() {
+	client, err := dapr.NewClient()
+	if err != nil {
+		panic(err)
+	}
+	CONFIGURATION_STORE := "configstore"
+	CONFIGURATION_ITEMS := []string{"appID1", "appID2"}
+	ctx, cancel := context.WithTimeout(context.Background(), 15 * time.Second)
+	defer cancel()
+
+	// Get configuration from config store
+	for _, item := range CONFIGURATION_ITEMS {
+		config, err := client.GetConfigurationItem(ctx, CONFIGURATION_STORE, item)
+		if err != nil {
```

Comment:
please also print the error or users to report or troubleshoot in case of a error.
Will the code continue to execute in case of error also?

Reply:
App should exit incase it is unable to retrieve config items. Updated code to do the same.